documentation/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -118,3 +118,187 @@ used to match no documents with the Lucene backend, but all documents with the E
 A boolean predicate with no clause will now consistently match no documents regardless of the backend.
 * API methods matching `*Async(..)` pattern (e.g. `SearchWorkspace#purgeAsync()`) will no longer block
 if internal queues of operations are full, but will throw `RejectedOperationException` instead.
+
+[[outboxpolling]]
+== Outbox polling system tables
+
```

Comment:
Wouldn't that fit the "Data format and schema changes" section? As a subsection, perhaps?

Reply:
That was where I wanted to put it ... but then I thought that that section is for index schema, so placed it as its own section 😄. Will move it!

Reply:
You're right, it was, but only because I didn't anticipate DB schema changes. It makes sense to just change the purpose of that section :)